tests/integration/pilot/istioctl_test.go

```diff
@@ -455,7 +456,7 @@ func TestXdsProxyStatus(t *testing.T) {
 
 			retry.UntilSuccessOrFail(t, func() error {
 				args := []string{
-					"proxy-status", fmt.Sprintf("%s.%s", podID, apps.Namespace.Name()),
+					"x", "proxy-status", fmt.Sprintf("%s.%s", podID, apps.Namespace.Name()), timeoutFlag,
```

Comment:
oof, does this mean we were no testing it this whole time?

Reply:
not sure...

Reply:
But when looking at some logs, weirdly "experimental" version of "proxy-status" seems to run in the recent tests. I suspect that somehow "PREFER-EXPERIMENTAL" was turned on in the test. https://github.com/istio/istio/blob/7bf52db38c91c89a4d56d6a94f402fd9ddaf6465/istioctl/cmd/root.go#L198
(previously, this part did not run with "experimental" version of "proxy-status")